### DISCUSSION TITLE
Option to disable budget rollover

### DISCUSSION THREAD

**Discussion by gforcedev** [ananthakumaran/paisa#84] [Ideas]
Really loving the budgeting system in Paisa!

With the way I'm using it, I'd prefer to not have the rollover shown and affecting the "amount available" - it would be amazing to have the option to disable that being taken into account in the calculations / the interface. Seems like it would be a relatively easy addition?

**Comment by ananthakumaran** [comment]
This is something I want as well. I will look into it soon.

**Reply by ananthakumaran** [reply]
Available on master https://github.com/ananthakumaran/paisa/commit/43b10938869fde1e86782bbb9e45fcbb14125fc7

**Reply by pashagolub** [reply]
Should this discussion be closed or marked as solved?

**Reply by ananthakumaran** [reply]
Looks like the marked as solved option is not available for this category (I don't see the button). In general, it's encouraged to mark the question as solved, but it's not mandatory. I treat discussions as a forum, whereas I am more strict about closing issues.

**Reply by pashagolub** [reply]
I see now. Indeed there is no such an option. My main concern was if one will look a new idea to work on, it would be great to have a filter/marker to know which proposals are implemented already.

Thanks one more time for your work!
